src/util/ChartUtils.js

```diff
@@ -884,6 +886,8 @@ export const parseSpecifiedDomain = (specifiedDomain, dataDomain, allowDataOverf
     const value = +MAX_VALUE_REG.exec(specifiedDomain[1])[1];
 
     domain[1] = dataDomain[1] + value;
+  } else if (toString.call(specifiedDomain[1]) === '[object Function]') {
```

Comment:
How about use _.isFunction to detect whether `specifiedDomain[1]` is a function?